src/main/java/dude/commands/Parser.java

```diff
@@ -0,0 +1,54 @@
+package dude.commands;
+
+import dude.task.ListManager;
+import dude.task.Todo;
+
+import java.util.List;
+
+public class Parser {
+    public static void parseInput(String input) {
+        String[] commands = input.split(" ",2);
+        int index;
+        switch (commands[0]) {
+
+        case "bye":
+            Interface.printBye();
+            System.exit(0);
+        case "todo":
+            ListManager.addTodo(commands[1]);
+            break;
+        case "deadline":
+            String[] deadline = commands[1].split("/by",2);
```

Comment:
coding standards: there should be space after comma
`String[] deadline = commands[1].split("/by", 2);`